schemathesis_hooks.py

```
@@ -0,0 +1,155 @@
+        FileNotFoundError: If the SQL file does not exist
+        IOError: If there are issues reading the file
+    """
+    # Regular expression to match UUIDs
+    uuid_pattern = re.compile(
+        r"\b[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}\b", re.IGNORECASE)
+
+    try:
+        with open(file_path, "r") as file:
+            sql_content = file.read()
+    except FileNotFoundError:
+        raise FileNotFoundError(f"SQL file not found: {file_path}")
```

Ruff (0.8.0)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)